### DISCUSSION TITLE
Link markdown not working in tag/release notes

### DISCUSSION THREAD

**Discussion by justclint** [community/community#22641] [New to GitHub]
<p>Hi, Im trying to add a link to our tag notes like:</p>
<pre><code>[View full changelog](https://github.com/my/repo/compare/v2.16.0...v2.17.0)
</code></pre>
<p>but the result come out like:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/ce1125099244fca88bc61bb28fb41a1f30af847d.png" data-download-href="/uploads/short-url/toX5ECTLRXgnKDELbxQNBqWFT7n.png?dl=1" title="Screen-Shot-2018-09-20-at-10.23.15-AM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/466984/181072254-ecf0741e-a06a-49fc-9d40-4abf6c816e7c.png" alt="Screen-Shot-2018-09-20-at-10.23.15-AM.png" data-base62-sha1="toX5ECTLRXgnKDELbxQNBqWFT7n" width="690" height="213" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/ce1125099244fca88bc61bb28fb41a1f30af847d_2_690x213.png, https://aws1.discourse-cdn.com/github/original/1X/ce1125099244fca88bc61bb28fb41a1f30af847d.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/ce1125099244fca88bc61bb28fb41a1f30af847d.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/ce1125099244fca88bc61bb28fb41a1f30af847d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen-Shot-2018-09-20-at-10.23.15-AM.png</span><span class="informations">974×302 50.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>The link does work but as you can see its not rendered properly.</p>
<p>If I can provide additional info that might be helpful, please let me know.</p>
<p>Thanks!</p>

**Comment by mpboom** [comment]
<p>Is the repo you are talking about public or do you know any public repo that has the same problem?</p>

**Comment by justclint** [comment]
<p>Thanks for the response <a class="mention" href="/mpboom">@mpboom</a>. The one in the OP is a private one but Ive just tested with a public one at <a href="https://github.com/justclint/vagrant-lemp-project-wrapper/releases" rel="nofollow noopener">https://github.com/justclint/vagrant-lemp-project-wrapper/releases</a> (see release v0.2.0) and its the same issue.</p>
<p>I also just noticed when you click on the release tag, on both private and public, the link is completely gone.</p>
<p>For example <a href="https://github.com/justclint/vagrant-lemp-project-wrapper/releases/tag/v0.2.0%C2%A0displays" rel="nofollow noopener">https://github.com/justclint/vagrant-lemp-project-wrapper/releases/tag/v0.2.0 displays</a> as:</p>
<p>“<a>View full changelog</a>” with no hyperlink.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>GitHub visualizes “releases” in two ways essentially. When you create a tag, GitHub shows that as a release but there isn’t a release object in GitHub’s database. If you want to create Markdown release notes for a release, you have to create a tag and then create a release object for it and add release notes. The tag that you linked to is annotated but that isn’t treated by GitHub as Markdown, only raw text. To show the difference:</p>
<ul>
<li>Your tag: <a href="https://github.com/justclint/vagrant-lemp-project-wrapper/releases/tag/v0.2.0" rel="nofollow noopener">https://github.com/justclint/vagrant-lemp-project-wrapper/releases/tag/v0.2.0</a>
</li>
<li>A release with associated notes based on a tag: <a href="https://github.com/atom/atom/releases/tag/v1.30.0" rel="nofollow noopener">https://github.com/atom/atom/releases/tag/v1.30.0</a>
</li>
</ul>
<p>You can find out more starting at <a href="https://help.github.com/articles/creating-releases/" rel="nofollow noopener">Creating releases</a> in our help docs.</p>
<p>I hope that helps!</p>

**Comment by justclint** [comment]
<p>Thanks <a class="mention" href="/lee-dohm">@lee-dohm</a>! That explains exactly what I was missing.</p>
